### PR TITLE
Saas 14.5 small fixes rtc aku

### PR DESCRIPTION
**Pull request by alexkuhn**

### CODE REVIEW

File: addons/mail/static/src/models/rtc/rtc.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ function factory(dependencies) {
              * connection that were established but failed or timed out.
              */
             browser.setInterval(() => {
-                this.channel_id && this.currentRtcSession && this._callSessions();
+                this.channelId && this.currentRtcSession && this._callSessions();
```
Comment (seb-odoo) [inline addons/mail/static/src/models/rtc/rtc.js:80]: Isn't the correct fix `this.channel` instead?

Comment (seb-odoo) [reply addons/mail/static/src/models/rtc/rtc.js:80]: Also the `setInterval` should be killed in `_willDelete`.

Comment (alexkuhn) [reply addons/mail/static/src/models/rtc/rtc.js]: changes done

File: addons/mail/static/src/models/rtc/rtc.js [hunk 2]
```diff
@@ -88,6 +88,7 @@ function factory(dependencies) {
             browser.removeEventListener('beforeunload', this._onBeforeUnload);
             browser.removeEventListener('keydown', this._onKeyDown);
             browser.removeEventListener('keyup', this._onKeyUp);
+            browser.clearTimeout(this._intervalId);
```
Comment (seb-odoo) [inline addons/mail/static/src/models/rtc/rtc.js:91]: `clearInterval`?

Comment (alexkuhn) [reply addons/mail/static/src/models/rtc/rtc.js:91]: indeed... ^^'

**Review by seb-odoo — APPROVED** [review]
@robodoo r+ rebase-ff

**Pull request closed without merging**